consensus/types/errors.go

```diff
@@ -83,8 +83,8 @@ func SendingMessage(msg *HotstuffMessage, nodeId NodeId) string {
 	return fmt.Sprintf("✉️ Sending message ✉️ to %d at (height, step, round) (%d, %d, %d)", nodeId, msg.Height, msg.Step, msg.Round)
 }
 
-func SendingStateSyncMessage(nodeId cryptoPocket.Address, height uint64) string {
```

Comment:
I think we need to rename this file from `errors.go` to `messages.go`

Reply:
makes sense, thanks.